backend/databasemodel/model.sql

```diff
@@ -1393,6 +1403,7 @@ CREATE TABLE kooste.osm_pisteet (
 	osm_id bigint NOT NULL,
```

Comment:
If we want to display information on the points, should we add `type_name` here that is "Pysäköintipaikka" or something like that?

Reply:
This was something I was going to your opinion at. One option would be to change the name of the osm_type field to type_name, if you think this field contains suitable information? The another option is to leave osm_type field as it is and add new field (called type_name and set its default value to something descriptive). Not sure however if Pysäköintipaikka is suitable since isn't it possible that this field contains both bus stops and parking places?

Reply:
No, the problem here is that osm_type refers to geometry type in OSM. OSM data model is different, so OSM type may be either "node", "way" or "rel". It cannot be anything else. We must have a separate field that tells us the tarmo category.

Reply:
Okay, that was a misleading field name then.. Since we already have a separate tarmo_category field with default value 'Pysäköinti' I don't really see any need for type_name field do let's just forget about it. Note that in lipas there is a point to store and show this information to user since type_name is much more accurate (e.g. kaukalo or frisbeegolfrata) than tarmo_category (e.g. Luistelu or Ulkoiluaktiviteetit)

Reply:
Okay perhaps there's no need for type_name apart from tarmo_category. All the osm data is available in the tags field anyway.

My idea was that there is a two-level categorization for Lipas objects. 1) tarmo_category and 2) under it, the exact type_name. I was thinking also parking and transit stops could have the same two levels.

But we can also leave the type_name empty. Then we will have to have "Pysäköinti", "Bussipysäkki", "Ratikkapysäkki" and "Rautatieasema" as separate tarmo_categories, so the user can distinguish different stops.

Reply:
By the way, how exactly can we give tarmo_category values Bussipysäkki, Ratikkapysäkki and Rautatieasema since these points are not stored in database tables?

Reply:
So my question is, what are the "ikoni ja kategoria" and "luistelukenttä" rows below for 1) parking lots and 2) transit stops?

<img width="496" alt="image" src="https://user-images.githubusercontent.com/9367712/159909241-94bb3f25-df11-4107-bba9-813b290db443.png">

Reply:
I have no idea. Maybe, just to be sure, we could keep this field type_name if we later decide that we want to use two-level categorization. Maybe I can set type_name's default value for osm_pisteet to 'Pysäköintipaikka' and for osm_alueet 'Pysäköintialue' and for both tables the default value for tarmo_category would be 'Pysäköinti'?